### PR TITLE
 Release 1.2.0

### PR DESCRIPTION
**Pull request by aaronjudd**
- React Router #2123 replaced flow-router and updated routing methods.
by @mikemurray
- PDP Cleanup - React Basic Details Card, Tax Card & Inventory Card #2086 by @kieha 
- Fix permissions for items shown in admin dashboard #2145 by @impactmass
- METEOR 1.4.4.2 
- Translation updates
- Revision Control Option Images 
#2236 by @abdulsemiu-atanda 
- Pin vsivsi:job-collection meteor package to 1.4.0 as a workaround to issue #2210 
#2228 by @hrath2015
- Add Product Name To Invoice 
#2226 by @abdulsemiu-atanda 
- Fix load order of shops/translations for Docker
#2224 by @jshimko
- Enhancement: Toggle Variant Visibility when Ancestor Product Visibility is Toggled
#2222 by @michael-alade
- Enhancement: Create Product Should Flip into Edit Mode
#2221 by @abdulsemiu-atanda 
- Fix Panel Expander Arrow Overlapping on Dropdown
#2211 by @abdulsemiu-atanda  
- Resolve Sent Email Closes Email Log Panel 
#2205 by @abdulsemiu-atanda 
- Added Kuwait, Saudi Arabia and Qatar currencies 
#2204 by @lcampanis
- Add OrderId To Checkout Completed Screen 
#2202 by @michael-alade
- Completed Order Logged Out State 
#2201, #1638 by @abdulsemiu-atanda 
- Fix Email Notification During Order Refund 
#2200 by @michael-alade
- made social share buttons on PDP ADA compliant 
#2196 by @minimart
- Add Reaction method for adding default roles 
#2174 by @spencern
- Order Summary not showing the correct data 
  #2207 by @kieckhafer
- Resolve Inconsistent Product Image of Cart Item 
#2115 by @abdulsemiu-atanda 
- Performance, pagination updates for orders by @kieha 
- Product and inventory fixes by @joykare

### CODE REVIEW

**Comment by kieckhafer** [comment]
Admin user: does not show prices on Product grid (logged out user / non-admin user is OK)

reaction

**Comment by kieckhafer** [comment]
Cannot change "Label" text on top-level variant
new_product
varient mov

File: server/api/core/addDefaultRoles.app-test.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
+import { Shops } from "/lib/collections";
+import { Reaction } from "/server/api";
+
+import { sinon } from "meteor/practicalmeteor:sinon";
```
Comment (zenweasel) [inline server/api/core/addDefaultRoles.app-test.js]: Import order

File: server/methods/core/orders.app-test.js [hunk 2]
```diff
@@ -0,0 +1,502 @@
+import { Meteor } from "meteor/meteor";
+import { Factory } from "meteor/dburles:factory";
+import accounting from "accounting-js";
+import { Reaction } from "/server/api";
+import { Orders, Products, Notifications, Media, Shops } from "/lib/collections";
+import { expect } from "meteor/practicalmeteor:chai";
+import { sinon } from "meteor/practicalmeteor:sinon";
+import Fixtures from "/server/imports/fixtures";
+// import { examplePaymentMethod } from "/server/imports/fixtures/packages";
```
Comment (zenweasel) [inline server/methods/core/orders.app-test.js]: Remove commented code

File: server/methods/core/orders.app-test.js [hunk 3]
```diff
@@ -0,0 +1,502 @@
+import { Meteor } from "meteor/meteor";
+import { Factory } from "meteor/dburles:factory";
+import accounting from "accounting-js";
```
Comment (zenweasel) [inline server/methods/core/orders.app-test.js]: Import order

File: imports/plugins/core/checkout/client/container/cartDrawerContainer.js [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+        }
+      });
+    }
+  }
+
+  handleRemoveItem(event) {
+    event.stopPropagation();
+    event.preventDefault();
+    const currentCartItemId = event.target.getAttribute("id");
+    $(`#${currentCartItemId}`).fadeOut(500, () => {
```
Comment (aaronjudd) [inline imports/plugins/core/checkout/client/container/cartDrawerContainer.js]: Jquery is exported globally by Meteor, but it's probably a good idea to import as that might not always be the case, but I feel like we'd rather just not use jquery,  I thought we were deprecating this syntax in favor of CSS only...

Comment (mikemurray) [inline imports/plugins/core/checkout/client/container/cartDrawerContainer.js]: Exactly. We should not be using jQuery, especially in React. We have react velocity for any animation needs.

File: imports/plugins/included/product-variant/client/containers/variantFormContainer.js [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+    const provider = Packages.findOne({
+      "shopId": shopId,
+      "registry.provides": "taxCodes",
+      "$where": function () {
+        const providerName = this.name.split("-")[1];
+        return this.settings[providerName].enabled;
+      }
+    });
+
+    if (provider) {
```
Comment (kieckhafer) [inline imports/plugins/included/product-variant/client/containers/variantFormContainer.js]: We want to leave this how it is for better / easier code readability.

**Comment by aaronjudd** [comment]
@mikemurray @kieckhafer @zenweasel is this ready -> seems to now have 3 pending reviews...

**Comment by zenweasel** [comment]
I'm still testing. I will approve as soon as I am done.

**Pull request merged**